external/dynamo/generalized/frontend.py

```
@@ -96,6 +96,17 @@ def __init__(self, runtime: DistributedRuntime) -> None:
         self.tokenizers: dict[str, AutoTokenizer] = {}
         # Regex to find one or more JSON objects optionally separated by semicolons
 
+        # Load model mapping from environment (model_name -> model_path)
+        # e.g., FRONTEND_MODEL_MAPPING='{"llama-3.3-70b": "/workspace/models/Llama-3.3-70B-Instruct"}'
+        self.model_mapping: dict[str, str] = {}
+        try:
+            mapping_str = os.environ.get("FRONTEND_MODEL_MAPPING", "{}")
+            self.model_mapping = json.loads(mapping_str)
+            if self.model_mapping:
+                logger.info("Loaded model mapping: %s", self.model_mapping)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)